libpod/runtime.go

```diff
@@ -97,7 +97,7 @@ func NewRuntime(options ...RuntimeOption) (runtime *Runtime, err error) {
 		if err != nil {
 			// Don't forcibly shut down
 			// We could be opening a store in use by another libpod
-			_, err2 := runtime.store.Shutdown(false)
+			_, err2 := store.Shutdown(false)
```

Comment:
Nit: Can you do this in a single line?

if  _, err2 := store.Shutdwon(false); err2 != nil